fluent-react/src/context.js

```diff
@@ -1,4 +1,4 @@
 import { createContext } from "react";
 import ReactLocalization from "./localization";
 
-export default createContext(new ReactLocalization([]));
+export default createContext(new ReactLocalization([], null));
```

Comment:
@Gregoor Do you think this is the right approach for the default context value?

Reply:
I think I'd initialize it with `null` in the constructor of `ReactLocalization` instead, imo that class is more concerned with that detail than the context.